index.css

```diff
@@ -1,5 +1,13 @@
 body {
```

Comment:
- [ ] In your PR description please add a more descriptive message highlighting the changes that you have made in this project and why/